docs/data-access/daco/applying.md

```diff
@@ -0,0 +1,106 @@
+
+### Application Progress Stages
+
+| Application Stage | Description | Application pdf | 
+| ----------------------- | --------------------------- | --------------------- | 
+| **Draft** | The application is in progress and is incomplete. <br /><br /> ![DACO Draft Progress](/assets/data-access/daco-draft.png)| You can download a **Draft pdf** that will display a watermark if you wish to use it to collaborate with others.<br /><br /> ![DACO Draft pdf](/assets/data-access/download-draft.png) |  
+| **Sign & Submit** | All required sections are completed and the application is ready to be signed by the Principal Investigator and Institutional Representative. <br /><br /> ![DACO Sign & Submit Progress](/assets/data-access/daco-signsubmit.png)| You can now download the **Finalized pdf** from the top button to collect the proper signatures.<br /><br /> ![DACO Finalized pdf](/assets/data-access/download-finalized.png)| 
+| **DACO Review** | Once you upload the signed application and click **Submit Application**, the application will be locked and it will be sent to the ICGC DACO for review. <br /><br /> ![DACO Review Progress](/assets/data-access/daco-review.png) | You can download the signed and submitted pdf using the **Signed pdf** top button. <br /><br />![DACO Signed pdf](/assets/data-access/download-signed.png)| 
+| **Revisions Requested** | If the ICGC DACO office requires revisions to your application before approval, you will receive an email with the details of what to revise and those sections of your application will be unlocked for editing. [Read more](/docs/data-access/daco/approval#requested-revisions).  <br /><br /> ![DACO Revisions Requested Progress](/assets/data-access/daco-revisions-requested.png)| You can download a **Draft pdf** that will display a watermark if you wish to use it to collaborate with others. <br /><br /> ![DACO Draft pdf](/assets/data-access/download-draft.png)| 
+| **Approved** | If the ICGC DACO approves your application, it will be viewable and some sections will remain editable. [Read more](/docs/data-access/daco/approval#approval-for-controlled-data-access).  <br /><br /> ![DACO Approved Progress](/assets/data-access/daco-approved.png)| You will be emailed the ICGC DACO approved copy of the application. <br /><br />  ![DACO Signed pdf](/assets/data-access/download-signed.png)| 
+
+
+### Applicant Dashboard
+
+When applications are in progress, the applicant dashboard will show the following:
+
+1. A card will show up on your dashboard tracking each application you have opened, what stage they are in, and provides quick links for editing. 
+1. Once an application is approved, the card will show the access expiry date and will eventually alert you when it’s almost time for renewal. 
+1. A smaller card in the top right of the dashboard will indicate if the logged in user has been approved for access to ICGC Controlled Data.  
+
+![DACO Dashboard](/assets/data-access/daco-dashboard.png)
```

Comment:
Same comment as above. Maybe we can move pic before the text?

Reply:
thanks! has been moved